mithril-common/src/chain_reader/pallas_chain_reader.rs

```diff
@@ -214,7 +216,10 @@ mod tests {
                 chainsync_server.recv_while_idle().await.unwrap();
 
                 if has_agency == HasAgency::No {
+                    println!("Has agency: No");
```

Comment:
should be these prints be really needed?

Reply:
No, I removed them :+1: